packages/cheqd-sdk/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@aries-framework/cheqd-sdk",
+  "main": "build/index",
+  "types": "build/index",
+  "version": "0.1.0",
+  "private": true,
```

Comment:
If we want to publish the package, we should remove this line. do you want to make adiitional prs before an alpha release is created?